src/framedecoration.h

```diff
@@ -15,7 +15,7 @@ class FrameDecorationData {
     bool visible = false;
     bool hasClients = false; // whether this frame holds clients at the moment
     bool hasParent = false;
-    Rectangle geometry;
+    Rectangle geometry = {};
```

Comment:
Wouldn't it be easier to add a default constructor in `Rectangle`?

Reply:
Not sure! I tried (just pushed) it, and curiously, it leads to `SnapData` needing its members initialized as well, because `Rectangle` is not a record type anymore.

These rules sure are funny.